apps/frontend/core-dapp/src/components/SettingsMenu/SettingsDropdown.tsx

```diff
@@ -31,6 +31,11 @@ const SettingsDropdown: React.FC = () => {
         trigger={['click']}
         placement="bottomRight"
         size="sm"
+        // Render even when hidden to:
+        // 1. Preload icons
+        // 2. Render IntercomButton which in turn may hide the default Intercom
+        //    widget
```

Comment:
and then this comment should be updated as it already seems like bad design choice that the visibility of Intercom widget is affected by SettingsDropdown component, but makes sense to keep it for preloading icons
```suggestion
        // force render to preload icons
```